### PR TITLE
Design About Page

### PR DESCRIPTION
**Pull request by wgranger**
This PR designs the main about page based on designs from inVision https://projects.invisionapp.com/d/main#/console/10495701/254604074/preview

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ module.exports = {
   devtool: 'eval-source-map',
   devServer: {
     contentBase: path.join(__dirname, '/src/'),
+    historyApiFallback: true,
```
Comment (shaunanoordin) [inline webpack.config.js]: Curious, what is this for? I found the [documentation](https://webpack.github.io/docs/webpack-dev-server.html#the-historyapifallback-option) but I'm not sure what the application is.

File: src/components/about/Index.jsx [hunk 2]
```diff
@@ -1,23 +1,107 @@
...
+            </div>
+
+          </div>
+        </div>
+        <div className="about-page__team">
+          <h2>The team</h2>
+          <div className="about-page__members">
+            {TheTeam.map((member) => {
+              return (
+                <div>
```
Comment (shaunanoordin) [inline src/components/about/Index.jsx]: [minor] React wants these divs to have the `key` attribute.

**Comment by shaunanoordin** [comment]
👍

File: webpack.config.js (hunk 1 above)
Comment (wgranger) [reply webpack.config.js]: I was having errors when navigating the site, I would often get a sort of "cannot GET /about..." type error, and this was a solution I found when browsing around.

**Comment by wgranger** [comment]
Ah yes, thanks for the sharp eye on that key issue.

File: webpack.config.js (hunk 1 above)
Comment (shaunanoordin) [reply webpack.config.js]: Oh Webpack, you are a mysterious mistress full of mystery.

**Pull request merged**